### PR TITLE
Css refactor and `Footer` components

### PR DESCRIPTION
**Pull request by chrichri17**
This PR does 2 things:
- Add a `Footer`component
 
  Screenshot 2022-11-08 at 14 26 58
- To make this work, I had to refactor a bit the css files by deleting a lot of `overflow` properties and writing normal css. I removed the scrollbar styles but i didn't mean to. We should keep it (i'll make the changes in a follow up PR)

Finally, I used normal fonteawesome library without a 3rd party. As you can see, it's just one line in the `index.html` file. However, this means that this file should be downloaded when the website starts loading and it might cause issues if the internet connection is low.

I did this because I couldn't find the social networks (facebook, insta, ...) icons in fortawesome.

If you think that this solution is easier, we can remove `fortawesome` in a follow up PR.

### CODE REVIEW

**Comment by chrichri17** [comment]
Current dependencies on/for this PR:
* main
  * **PR #5** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/5" target="_blank">Graphite</a> 
    * **PR #6** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/6" target="_blank">Graphite</a> 
      * **PR #7** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/7" target="_blank">Graphite</a>  👈
        * **PR #8** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/8" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/YacineSteeve/quizoot/7?utm_source=stack-comment).

File: frontend/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <head>
   <meta charset="UTF-8" />
   <link rel="icon" href="/icon-no-background.svg" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (chrichri17) [inline frontend/index.html]: Just one line to include fonteawesome

File: frontend/src/components/Footer.vue [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            </div>
+            <ul class="copyright">
+                <li>&copy; Fondation Vallet</li>
+                <li>
+                    <a target="_blank" rel="noreferrer" href="https://www.fondationvallet.org/eeia/">
+                        École d'Été en Intelligence Artificielle (EEIA)
+                    </a>
+                </li>
+                <li>
+                    <a target="_blank" rel="noreferrer" href="#i">
```
Comment (chrichri17) [inline frontend/src/components/Footer.vue:29]: I couldn't find a good link for this one

File: frontend/src/components/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    border: none;
+}
+
+.copyright li {
+    display: inline-block;
+    border-left: solid 1px rgba(144, 144, 144, 0.25);
+    padding: 15px;
+}
+
+@media only screen and (max-width: 600px) {
```
Comment (chrichri17) [inline frontend/src/components/Footer.vue:82]: This is how you do mobile version. 

Screenshot 2022-11-08 at 14 25 18

File: frontend/src/components/Main.vue [hunk 4]
```diff
@@ -11,29 +11,12 @@
...
 .main-container {
     width: 100%;
-    height: 90vh;
+    height: auto;
     margin-top: 10vh;
     background-color: var(--background-color);
-    overflow: scroll;
-}
-
-.main-container::-webkit-scrollbar {
```
Comment (chrichri17) [inline frontend/src/components/Main.vue:20]: I should revert this change and keep the styles for the scrollbar

File: frontend/src/components/QuizCard.vue [hunk 5]
```diff
@@ -46,6 +46,7 @@ const bgColor = computed(() => (quiz.status ? '#008000' : '#dfe2ec'));
 
 .quiz-preview-container:hover {
     transform: scale(1.05);
+    cursor: pointer;
```
Comment (YacineSteeve) [inline frontend/src/components/QuizCard.vue:49]: 👍🏾 Nice

File: frontend/src/components/Main.vue (hunk 4 above)
Comment (YacineSteeve) [reply frontend/src/components/Main.vue:20]: Okay

File: frontend/index.html (hunk 1 above)
Comment (YacineSteeve) [reply frontend/index.html]: See https://fontawesome.com/search?o=r&m=free&f=brands for social networks icons.

For example, Facebook's can be used like this `<font-awesome-icon icon="fa-brands fa-facebook" />` 
after being imported `import { faFacebook } from '@fortawesome/free-brands-svg-icons'` .

(`@fortawesome/free-brands-svg-icons` already in `package.json`)

File: frontend/src/components/Footer.vue (hunk 2 above)
Comment (YacineSteeve) [reply frontend/src/components/Footer.vue:29]: Maybe the LinkedIn profile again ?

File: frontend/src/components/Footer.vue [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    <div id="footer">
+        <div class="container">
+            <div class="icons">
+                <a class="icon" href="https://fr-fr.facebook.com/BeninExcellence/">
+                    <i class="fa fa-facebook-f" />
+                </a>
+                <a class="icon" href="https://twitter.com/excellencebenin">
+                    <i class="fa fa-twitter" />
+                </a>
+                <a class="icon" href="#i">
```
Comment (YacineSteeve) [inline frontend/src/components/Footer.vue:13]: Is it the [same problem](https://github.com/YacineSteeve/quizoot/pull/7/files?diff=unified&w=0#r1016629224) here ?

**Review by YacineSteeve — APPROVED** [review]
Everything seems okay. As potential issues have already been highlighted, we could work on them after.

File: frontend/index.html (hunk 1 above)
Comment (chrichri17) [reply frontend/index.html]: Awesome. I'll make the changes then

File: frontend/src/components/Footer.vue (hunk 6 above)
Comment (chrichri17) [reply frontend/src/components/Footer.vue:13]: Oh I think I just forgot this one
